summary/views.py

```diff
@@ -40,65 +39,53 @@
 from summary.models import UserUnlinkedRuns
 
 
-def add_user_uuid(request):
+def add_user_uuid(request, user_uuid, run_uuid):
```

Comment:
request param no longer needed?

Reply:
wouldn't you also need to change the arguments being passed wherever this function is called? I'm not seeing that.

Reply:
The `views.py` functions are used by Django as the endpoints and have to implement `request` as the first argument. https://docs.djangoproject.com/en/3.2/topics/http/views/